packages/lesswrong/server/recommendations.js

```diff
@@ -159,18 +160,78 @@ const getRecommendations = async ({count, algorithm, currentUser}) => {
   }
 };
 
+const getResumeSequences = async (currentUser, context) => {
+  if (!currentUser)
+    return [];
+  if (!currentUser.partiallyReadSequences)
+    return [];
+  
+  return Promise.all(_.map(currentUser.partiallyReadSequences,
+    async partiallyReadSequence => ({
+      sequence: partiallyReadSequence.sequenceId
+        ? await context["Sequences"].loader.load(partiallyReadSequence.sequenceId)
```

Comment:
I think I prefer this to a proper function instead of an object construction, but it's fine this way. Just seemed a bit hard to read.